src/GraphQL/Query.php

```diff
@@ -37,8 +37,9 @@ public function __construct(EntityManagerInterface $em, TokenStorageInterface $t
     public function current(bool $loggedIn = false): array
     {
         $groups = [];
-        if ($loggedIn && $user = $this->user()) {
-            $groups = $user->getRelations()->toArray();
+        if ($loggedIn) {
+            assert($this->user() !== null);
```

Comment:
This assert is not guaranteed to be always true. Instead, please change the code to:
```php
if ($loggedIn && null !== $user = $this->user()) {
```